roles/base/tasks/redhat_tasks.yml

```diff
@@ -10,6 +10,7 @@
     update_cache: true
     state: latest
   with_items:
+    - gcc
```

Comment:
@erikh is this due the issue that we saw with gcc not being packaged in packer boxes? It should be fixed in new boxes (https://github.com/contiv/build/pull/60), can you try the new box `0.3.6ba5a75` ?